src/init.cpp

```diff
@@ -1054,7 +1058,7 @@ bool AppInitParameterInteraction()
     if (gArgs.IsArgSet("-minimumchainwork")) {
         const std::string minChainWorkStr = gArgs.GetArg("-minimumchainwork", "");
         if (!IsHexNumber(minChainWorkStr)) {
-            return InitError(strprintf("Invalid non-hex (%s) minimum chain work value specified", minChainWorkStr));
+            return InitError(strprintf(_("Invalid non-hex (%s) minimum chain work value specified"), minChainWorkStr));
```

Comment:
This change as a whole seems to be adding a lot of new translated strings. But I wonder if it would be better to be more conservative and not add new translations here, instead saving that part for a followup PR and selectively leaving less common errors untranslated. One way to do this might be with a `DoNotTranslate` function

```c++
bilingual_str DoNotTranslate(std::string original)
{
    return {original, original};
}
```

Reply:
`DoNotTranslate()` has been implemented.

Reply:
Nit: don't forget to actually use `Untranslated ` here.